sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClient.java

```diff
@@ -1131,7 +1141,7 @@ public Mono<MsalToken> authenticateWithSharedTokenCache(TokenRequestContext requ
      * @param request the details of the token request
      * @return a Publisher that emits an AccessToken
      */
-    public Mono<AccessToken> authenticateToArcManagedIdentityEndpoint(String identityEndpoint,
+    Mono<AccessToken> authenticateToArcManagedIdentityEndpoint(String identityEndpoint,
```

Comment:
default visibility is `package-private` which is not what I think we want here?

Reply:
made private.
package private helps to keep things accessible in tests, but tests does not use these directly currently.